server/notification-providers/pushover.js

```diff
@@ -16,6 +16,7 @@ class Pushover extends NotificationProvider {
             "sound": notification.pushoversounds,
             "priority": notification.pushoverpriority,
             "title": notification.pushovertitle,
+            "ttl": notification.pushoverttl,
```

Comment:
Since `notification.pushoverttl` is `undefined` for old notification setups, is it safe?